src/components/Calculator/ScoreTable.js

```diff
@@ -1,15 +1,27 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
+import Threshold from './Threshold'
 import getCourseScore from '../../utils/getCourseScore'
 import facultiesArray from '../../faculties.json'
+import thresholdsArray from '../../thresholds.json'
 
 const ScoreTable = ({ subjects }) => (
   <div>
     <br />
     <br />
     <table className='table table-bordered score-table'>
       <tbody>
+
+        <tr>
+          <th />
+          <th />
+          <th />
+          <th colSpan='4' className='thresholds-title'>
```

Comment:
If you implement suggestion "What about getting available years from thresholds.json" then this colSpan needs to be computed accordingly.